bleak/backends/bluezdbus/scanner.py

```diff
@@ -242,7 +242,10 @@ def _parse_msg(self, message: Message):
             # discovered devices list
 
             msg_path = message.body[0]
-            del self._devices[msg_path]
+            try:
```

Comment:
FWIW, here is how I fixed it: https://github.com/hbldh/bleak/pull/389/commits/5bf7266825c7b4714b90e59200739bfc8d014018

Reply:
But nothing wrong with this way either :smile:

Reply:
👍 I like it.  These were a bit old, I just wanted to document them since they were all short and sweet.